File: web3rpc/rpc-specs/code-samples/python/debug/runtimeTracing/startGoTrace.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

fileName = "go.trace"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.start_go_trace(fileName)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/runtimeTracing/startGoTrace.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "go.trace";

    provider.debug.startGoTrace(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()
